libgraph/test/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Keep alphabetical order
```

Comment:
I respect that goal. But it's not realistic unless it's checked in CI. Maybe we could have a sort checker we apply to a few selected files.